shoov/modules/custom/shoov_ui_build/shoov_ui_build.module

```diff
@@ -64,3 +71,50 @@ function shoov_ui_build_create_pr($node) {
 
   return drupal_http_request($url);
 }
+
+/**
+ * Trigger "New build" event when bew ui_build is created.
+ *
+ * Implements hook_node_insert().
+ */
+function shoov_ui_build_node_insert($node) {
+  if ($node->type != 'ui_build') {
+    return;
+  }
+
+  $wrapper = entity_metadata_wrapper('node', $node);
+  $repo_id = $wrapper->og_repo->value(array('identifier' => TRUE));
+
+  // todo: Discover how to check actual user's access, not admin.
+  $account = user_load($node->uid);
+  $handler = restful_get_restful_handler('builds');
+  $handler->setAccount($account);
+  $result = $handler->get($node->nid);
+
+  $data = array($result[0]);
+  shoov_pusher_trigger_event($repo_id, 'new_ui_build', $data);
+}
+
+/**
+ * Implements hook_node_access().
+ *
+ * Grant a user to see their own published node. This trivial access is needed
```

Comment:
@HelenaEksler ker fyi, this is the "hack" needed to grant access, instead of the `user_load(1)`